spec/feature/queue/case_details/edit_issues_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "support/vacols_database_cleaner"
+require "rails_helper"
+
+RSpec.feature "Edit Issues", :all_dbs do
+  before do
+    FeatureToggle.enable!(:allow_judge_edit_issues)
+  end
+
```

Comment:
```suggestion

after do
  FeatureToggle.disable!(:allow_judge_edit_issues)
end

```

Don't know how important it is, but there's a pattern of disabling flags after your tests have run.